src/Markdown.js

```diff
@@ -175,14 +175,6 @@ export default class Markdown {
         const renderer = new commonmark.HtmlRenderer({safe: false});
         const real_paragraph = renderer.paragraph;
 
-        // The default `out` function only sends the input through an XML
-        // escaping function, which causes messages to be entity encoded,
-        // which we don't want in this case.
-        renderer.out = function(s) {
-            // The `lit` function adds a string literal to the output buffer.
-            this.lit(s);
-        };
-
```

Comment:
This is the `out` function it's overriding:
```JavaScript
function out(s) {
  this.lit(this.esc(s, false));
}
```
and `esc`:
```JavaScript
function(s, preserve_entities) {
    if (reXmlSpecial.test(s)) {
        if (preserve_entities) {
            return s.replace(reXmlSpecialOrEntity, replaceUnsafeChar);
        } else {
            return s.replace(reXmlSpecial, replaceUnsafeChar);
        }
    } else {
        return s;
    }
}
```

Removing it accounts for [this problem](https://github.com/matrix-org/matrix-react-sdk/pull/4008#issuecomment-590282297). Since `toPlaintext` emits HTML (rather unintuitively, given the name), if we don't encode HTML entities they are processed literally.

As an example, if the `out` override is left in, the message `\*hello\* world '<'` returns a `formatted_body` of `*hello* world '<'`. Since the message is HTML, Riot produces this:

![2020-06-05-13:14:07](https://user-images.githubusercontent.com/14955039/83909456-7286ea00-a72e-11ea-8e39-73afb33c6be0.png)